package.json

```diff
@@ -25,7 +25,7 @@
     "release:patch:all": "yarn build:all && yarn validate && yarn verbump:patch:all && yarn publish:release",
     "release:prerelease-bump:all": "yarn build:all && yarn validate && yarn verbump:prerelease:all && yarn publish:release",
     "release:prerelease:all": "yarn build:all && yarn validate && yarn publish:release",
-    "storybook": "yarn watch:foundations & yarn watch:icons & start-storybook",
+    "storybook": "start-storybook",
```

Comment:
The reason this is like this is because all components depend on `@guardian/src-foundations`, and some depend on `@guardian/src-icons`. This means changes to foundations are not picked up by Storybook, because the components depend on the the `dist` code under src/core/foundations. Does that make sense? Maybe there's some way of fixing this with path remapping somewhere.